src/transformers/file_utils.py

```diff
@@ -548,3 +548,18 @@ def wrapper(*args, **kwargs):
             raise ImportError(f"Method `{func.__name__}` requires TF.")
 
     return wrapper
+
+
+def unzip(zip_path, dest_dir):
```

Comment:
add type hints to functions in file_utils

Reply:
Done!